### ISSUE TITLE
Latest Frost crashes Lawnchair.

### ISSUE THREAD

**Issue by marcdw1289** [dkanada/frost#223]
I was jazzed to have my icon requests filled. Alas, when trying to customize shortcuts after updating Frost, launcher keeps crashing when I choose Frost. Lawnchair automatically logs the problem for me.
Main error being...
java.lang.ArithmeticException: divide by zero

Frost 1.6.2
Lawnchair 2.0-2551
AOSP-CAF, Android 7.1.2
Motorola Moto G5 Plus XT1687.

Stacktrace logged by Lawnchair:
https://del.dog/odadoxitas

**Comment by marcdw1289** [comment]
Hmm, now I'm not sure about anything. I had updated Lawnchair and Frost in the same session. Went to previous version of the launcher with same results. But also tried Frost 1.6.0 and 1.6.1. The launcher crashes. Did all the requisite data and cache clearing.
So strange as everything was fine or I wouldn't be using the pack. Maybe something on my end somewhere. Will have to check on other ROMs and devices.

**Comment by h-bruch** [comment]
Have you tried to downgrade to the previous version of lawnchair _and_ frost? Your new icons are not in 1.6.2 anyway, but will only appear in the next release.

Have you asked the people @lawnchair (they can probably pinpoint the problem easier/tell if it originates even from frost).

Cheers!

**Comment by marcdw1289** [comment]
Haha, yeah, I realized my mistake regarding the new icons. Oops.
Yeah, I'll check Lawnchair's Telegram group and whatnot to see if it's a known issue. Thanks.

**Comment by dkanada** [comment]
Have you been able to find a solution?

**Comment by marcdw1289** [comment]
Oh, so sorry for not following up. No, no solution.
After the original posts I checked Lawnchair's github. An identical issue was posted back in September. Safe to assume it got lost among all the other issues but I added a "me too" response anyway.

Today when setting up another Google-free setup I found the same thing happens with Ameixa icon pack on Lawnchair v2. I don't know if it's the dashboard used by some icon packs or something else.

https://github.com/LawnchairLauncher/Lawnchair/issues/1737
